stl/inc/yvals_core.h

```diff
@@ -140,6 +140,11 @@
 // P0618R0 Deprecating <codecvt>
 // Other C++17 deprecation warnings
 
+// Implemented when char8_t is available (C++14/17 with /Zc:char8_t, C++20 without /Zc:char8_t-):
+// P0482R6 Library Support For char8_t
+//     (mbrtoc8 and c8rtomb not yet implemented, see GH-2207)
+// P1423R3 char8_t Backward Compatibility Remediation

```

Comment:
Should we say P1423R3 is partially implemented in C++14/17 with `char8_t` modes? As some `operator<<` overloads (except for those of `char8_t`) are only deleted since C++20.

Reply:
That's a *very* interesting question. This comment matches what the feature-test macro is doing, but you're correct about what the implementation is actually doing. (As an extra complication, the implementation is affected by `_HAS_STREAM_INSERTION_OPERATORS_DELETED_IN_CXX20`, but the feature-test macro is not.)

Adding a `(partially implemented in C++14/17)` note would be reasonable - I think that trying to describe the behavior further would be a lot of effort for diminishing returns.

Reply:
Thinking about it further, it's probably best to move P1423R3 back to the C++20 section, as the bulk of the deleted overloads (and the ones that users are most likely to notice) are controlled by C++20 with the escape hatch.

Thanks for noticing this!